docs/source/1.0/spec/aws/aws-cloudformation.rst

```diff
@@ -0,0 +1,825 @@
+    * - ``password``
+      - Write only
+      - + ``@mutability`` trait specified in ``FooProperties`` via
+          ``additionalSchemas``.
+    * - ``readProperty``
+      - Read only
+      - + Returned in the ``read`` lifecycle via ``GetFooResponse``.
+    * - ``secret``
+      - Write only
+      - + ``@mutability`` trait specified in ``CreateFooRequest``.
+    * - ``tags``
+      - Full
+      - + ``@mutability`` trait specified in ``CreateFooRequest``.
+    * - ``updatedAt``
+      - Read only
+      - + ``@mutability`` trait specified in ``GetFooResponse``.
+    * - ``writeProperty``
+      - Write only
+      - + Specified in the ``create`` lifecycle via ``CreateFooRequest``.
+        + Specified in the ``update`` lifecycle via ``UpdateFooRequest``.
+
+
+.. _aws.cloudformation#cloudFormationName-trait:
+
+-----------------------------------------------
+``aws.cloudformation#cloudFormationName`` trait
+-----------------------------------------------
+
+Summary
+    The cloudFormationName trait allows a CloudFormation `resource property`__
+    name to differ from a structure member name used in the model.
+Trait selector
+    ``structure > member``
+
+    *Any structure member*
+Value type
+    ``string``
+
+.. __: https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-schema.html#schema-properties-properties
+
+Given the following structure definition that is converted to a CloudFormation
+resource:
+
+.. tabs::
+
+    .. code-tab:: smithy
+
+        namespace smithy.example
+
+        use aws.cloudformation#cloudFormationName
+
+        structure AdditionalFooProperties {
+            bar: String,
+
+            @cloudFormationName("Tags")
+            tagList: TagList,
+        }
+
+the CloudFormation resource would have the following property names derived
+from it:
+
+::
+
+    "bar"
+    "Tags"
+
+.. _aws.cloudformation#additionalIdentifier-trait:
+
+-------------------------------------------------
+``aws.cloudformation#additionalIdentifier`` trait
+-------------------------------------------------
+
+Summary
+    Indicates that the CloudFormation property generated from this member is an
+    `additional identifier`__ for the resource.
+Trait selector
+    ``structure > :test(member > string)``
+
+    *Any structure member that targets a string*
+Value type
+    Annotation trait
+
+.. __: https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-schema.html#schema-properties-additionalidentifiers
+
+Each ``additionalIdentifier`` uniquely identifies an instance of the
+CloudFormation resource it is a part of. This is useful for resources that
+provide identifier aliases.
+
+``additionalIdentifier`` traits are ignored when applied outside of the input
+to an operation bound to the ``read`` lifecycle of a resource. The
+``additionalIdentifier`` trait MUST NOT be applied to members with the
+:ref:`aws.cloudformation#mutability-trait` set to ``write-only``.
```

Comment:
I've moved this sentence to a "Validation" section in the definition, as it cannot be done with a conflicts property since it validates against the value of the trait, not the presence.